ngi_pipeline/engines/piper_ngi/local_process_tracking.py

```diff
@@ -198,8 +197,10 @@ def update_charon_with_local_jobs_status(quiet=False, config=None, config_file_p
                         set_status = "UNDER_ANALYSIS"
                         if workflow == "merge_process_variantcall":
                             status_field = "alignment_status"
+                            recurse_status = "RUNNING"
```

Comment:
Here again is an annoying inconsistency. For `analysis_status`, it's `RUNNING` for seqrun records but `UNDER_ANALYSIS` for sample records. It is `UNDER_ANALYSIS` for both samples and seqruns with regard to the `genotype_status` field, which is nice.